services/monitor/src/ChainMonitor.ts

```diff
@@ -31,6 +31,7 @@ export class ChainMonitor extends EventEmitter {
   private blockIntervalLowerLimit: number;
   private bytecodeInterval: number;
   private bytecodeNumberOfTries: number;
+  private running = false;
```

Comment:
I see that this variable is set at start and stop, is it needed to set it to false also when there is an error?

Reply:
Not sure but I don't think so.